join-chat-to-teams-meeting/Chat Experience with File Sharing/package.json

```diff
@@ -11,11 +11,11 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@azure/communication-calling": "^1.1.0",
-    "@azure/communication-chat": "1.3.2-beta.2",
-    "@azure/communication-common": "^1.1.0",
-    "@azure/communication-identity": "^1.0.0",
-    "@azure/communication-signaling": "1.0.0-beta.18"
+    "@azure/communication-calling": "^1.17.1",
+    "@azure/communication-chat": "^1.3.2",
```

Comment:
Should we be unpinning these depenendencies?

Reply:
> unpinning these dependencies

Seems like a bigger topic. Should we discuss this in Monday's standup?